.github/workflows/storybook.yml

```diff
@@ -43,4 +43,4 @@ jobs:
                   token: ${{ secrets.GITHUB_TOKEN }}
                   projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
                   autoAcceptChanges: "main"
-                  onlyStoryFiles: "**/*regression.stories.tsx"
```

Comment:
😞  https://github.com/chromaui/chromatic-cli/issues/721

Reply:
Yeah, I can't seem to get this to work. The `chromaui` GitHub action logs something like "stories will be limited to these files" indicating that it understands the option, but then all the stories get uploaded anyway. And then no tests run.

Somehow, WB is getting snapshot testing to work. And setting it up for the mathjax-renderer was pretty painless. I wonder if maybe the fact that we're running Storybook through Vite is getting in our way?